### PR TITLE
Bugfix: Run translation check on PR

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Runs translation check not only on push, but also on each PR. This was a bugfix which went forgotten.

CHANGELOG: SKIP

**Context: Fixes an issue?**

Fixes: #2677 

**Does this change need documentation? What needs to be documented and how?**

No

**Status of this Pull Request**

To be reviewed

**What is missing until this pull request can be merged?**

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

**Comment by ann0see** [comment]
See https://github.com/ann0see/jamulus/pull/78 as test. Unfortunately the verification now runs twice.

File: .github/workflows/translation-check.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ name: Verify translations
 
 on:
   workflow_dispatch:
+  pull_request:
+    paths:
+      - 'src/translation/**'
```
Comment (hoffie) [inline .github/workflows/translation-check.yml:7]: This path (and the one below) could theoretically be even more specific as long as the other check is disabled.
PRs which change the script or the workflow itself should probably also run it.

```suggestion
      - 'src/translation/wininstaller/**'
      - 'tools/check-wininstaller-translations.sh'
      - '.github/workflows/translation-check.yml'
```

**Review by hoffie — APPROVED** [review]
> Unfortunately the verification now runs twice.

I don't think this can be avoided. Usually, one of the runs is in the source repo (`push`) and one in the target repo (`pull_request`, although running with a token of the source repo...).

File: .github/workflows/translation-check.yml (hunk 1 above)
Comment (ann0see) [reply .github/workflows/translation-check.yml:7]: Agree.

Comment (hoffie) [reply .github/workflows/translation-check.yml]: Could probably also be done for the other path list for `push`?

**Review by pljones — APPROVED** [review]
Approving on basis the build is clean.  And @hoffie is happy 😃.

**Comment by ann0see** [comment]
I'll have a look at @hoffie's other comment tomorrow

**Pull request merged**